task/show-sbom-rhdh/0.1/show-sbom-rhdh.yaml

```diff
@@ -40,7 +40,7 @@ spec:
     - name: IMAGE_URL
       value: $(params.IMAGE_URL)
     script: |
-      #!/busybox/sh
+      #!/bin/bash
```

Comment:
Just my 2c: `/bin/sh`  might be a better choice in general, but I'd double check with `shellcheck`.